portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,79 @@
 <html>
```

Comment:
GitHub logistics:  Can you keep one set of changes to one pull request?  If we are reviewing, and then you make changes but the changes are in a new pull request we can't see the diff and the conversation gets hidden.

Reply:
My mistake. I'll be sure to do that from here on out.